tests/test_cli/test_generate/test_protocols.py

```diff
@@ -72,13 +72,13 @@ def test_exit_code_equal_to_0(self):
     def test_resource_folder_contains_module_message(self):
         """Test that the protocol folder contains message.py module."""
         p = Path(self.t, self.agent_name, "protocols", "two_party_negotiation", "message.py")
-        original = Path(CUR_PATH, "data", "generator", "two_party_negotiation", "message.py")
+        original = Path(CUR_PATH, "..", "examples", "protocol_specification_ex", "output", "two_party_negotiation", "message.py")
         assert filecmp.cmp(p, original)
 
     def test_resource_folder_contains_module_protocol(self):
         """Test that the protocol folder contains serialization.py module."""
         p = Path(self.t, self.agent_name, "protocols", "two_party_negotiation", "serialization.py")
-        original = Path(CUR_PATH, "data", "generator", "two_party_negotiation", "serialization.py")
+        original = Path(CUR_PATH, "..", "examples", "protocol_specification_ex", "output", "two_party_negotiation", "serialization.py")
```

Comment:
I moved the generated protocols out of the `tests/` directory. Otherwise coverage loads the files and then throws an error because it cannot find the modules.